### DISCUSSION TITLE
import and start does not work

### DISCUSSION THREAD

**Discussion by 2021florian** [jhuckaby/Cronicle#422] [Q&A]
Hello,
I have a problem with cronicle.

When I start the server (container) for the first time and then ...
"/opt/cronicle/bin/control.sh setup".
"/opt/cronicle/bin/control.sh start"
everything works.

But now I have backups, which I import into the freshly started container, but then cronicle hangs at "Waiting for master server..." and nothing happens.

"/opt/cronicle/bin/control.sh import /cronicle_restore/cronicle_20210707_1551.txt"
"/opt/cronicle/bin/control.sh start"

Do any of you have any idea why this is not working?

cronicle version 0.8.62
node v12.18.3
Container (debian 9.13)

Thanks for your help.
Florian

**Comment by mikeTWC1984** [comment]
That should be because you server info from old to a new host. When you initially setup cronicle build script adds current host (the old one) to the master list. When you restore on the new server this is not happening anymore, so new host is not becoming the master. To fix it you need to edit master group regex (like do smth like "storage-cli.js edit server_groups/0", maybe check docs how to do that), or just remove lines starting with "server" on the backup file before restoring.

**Comment by 2021florian** [comment]
Hello mikeTWC1984,
thank you very much for your detailed answer, that solved my problem. I was not aware when reading the documentation that this information is contained in the backup, but also indirectly depends on the container, because it gets a different name and IP each time.

Many greetings
Florian

```
# Start Container
/opt/cronicle/bin/control.sh setup
sed -i '/^global\/server_groups -/d' ${backup_filename} 
sed -i '/^global\/server_groups\/0 -/d' ${backup_filename}
sed -i '/^global\/servers -/d' ${backup_filename} 
sed -i '/^global\/servers\/0 -/d' ${backup_filename}
/opt/cronicle/bin/control.sh import ${backup_filename}
/opt/cronicle/bin/control.sh start
```
